operator/params.yaml

```
@@ -701,3 +701,23 @@ JVM_OPT_MAX_GC_PAUSE_MILLIS:
 JVM_OPT_G1R_SET_UPDATING_PAUSE_TIME_PERCENT:
   description: "Have the JVM do less remembered set work during STW, instead preferring concurrent GC. Reduces p99.9 latency."
   default:
+
+READINESS_CHECK_INITIAL_DELAY:
+  description: "An amount of time in seconds to wait before starting the readiness check attempts."
```

Let's use the [k8s terminology](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes)

```suggestion
  description: "Number of seconds after the container has started before the readiness check is initiated."
```